libs/devices/harmony/models/src/lib/endpoints/file-system.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { z } from "zod"
+
+export enum HarmonyDirectory {
+  Relaxation = "/user/media/app/relaxation",
+  Alarm = "/user/media/app/alarm",
+}
+
+export const HarmonyFileSchema = z.object({
+  path: z.string(),
```

Comment:
Could this be an enum of `HarmonyDirectory`?

```suggestion
  path: z.enum(HarmonyDirectory),
```